politeiawww/api/v1/v1.go

```diff
@@ -540,6 +545,25 @@ type VerifyUserPaymentReply struct {
 	PaywallTxNotBefore int64  `json:"paywalltxnotbefore"` // Minimum timestamp for paywall tx
 }
 
+// Users is used to request a list of users given a filter.
+type Users struct {
```

Comment:
Instead of having both Username and Email fields, do you think it would make more sense to just pass up a single Filter field and then try to match that against the email and username for each user?

Reply:
Makes sense.